.github/workflows/build.yml

```diff
@@ -76,6 +76,32 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions-rust-lang/setup-rust-toolchain@v1
 
+      # Install slow-odgi.
+      - uses: actions/cache@v4
+        id: cache-uv
+        with:
+          path: ~/.cache/uv
+          key: ${{ runner.os }}-python-${{ matrix.python-version }}-uv
+      - name: Create and activate virtualenv with uv
+        run: |
+          curl -LsSf https://astral.sh/uv/install.sh | sh
+          uv venv
+          echo "VIRTUAL_ENV=.venv" >> $GITHUB_ENV
+          echo "$PWD/.venv/bin" >> $GITHUB_PATH
+      - name: Install Python tools
+        run: uv pip install -r requirements.txt
+
+      # Install odgi
+      - name: Pull odgi container
+        run: |
+          docker pull quay.io/biocontainers/odgi:0.8.3--py310h6cc9453_0
+          docker tag quay.io/biocontainers/odgi:0.8.3--py310h6cc9453_0 odgi
```

Comment:
Maybe we should make these odgi versions match for the two jobs?

Reply:
Done! Changed this to the newer version